src/tesla_ver/bubble_chart/bubble_chart_layout.py

```diff
@@ -64,7 +73,7 @@
                                     id="y_dropdown",
                                     options=[],
                                     value=None,
-                                    placeholder="Y Axis Values",
+                                    placeholder="Choose an Taxa for the Y Axis",
```

Comment:
Taxa doesn't need to be capitalized:
"choose a taxa ...."

Reply:
addressed in b5d1e7f